spec.html

```diff
@@ -276,6 +276,7 @@ <h1>ArrayBuffer.prototype.resize ( _newLength_ )</h1>
         1. If IsDetachedBuffer(_O_) is *true*, throw a *TypeError* exception.
```

Comment:
```suggestion
```

Reply:
In that case, given lack of interop right now between V8 and JSC, I'd prefer to sink the detached check to be after the `_newLength_` coercion. I'll add it to the agenda.